### DISCUSSION TITLE
GTR 4 MINI 1.0.1 emulator does not run on API 2

### DISCUSSION THREAD

**Discussion by XiaomaiTX** [zepp-health/zeppos-forum#132] [Developer Feedback]
Some time ago I gave feedback that the GTR 4 MINI device (version 1.0.0) in ZeppOS emulator did not start properly(https://github.com/orgs/zepp-health/discussions/126), this problem was recently fixed by Huami, but in the new version 1.0.1 I found that the GTR 4 MINI could not be started when creating the ZeppOS API 2 applet, prompting "Warning Unsupported targets(416x416-gtr-4-mini) have been removed ."

The configuration in app.json has been verified to be available in API 1

Link to download the code that went wrong

https://iseei.cn/d/Personal/OSS-US/guest/gtrmini2test.zip?sign=WsbVVWHFZiv93d6XrOLhNQnrTR3HOyVlCcDQ9Wx8u9s=:0

Other information is as follows

![-30d14f170e25f43a.jpg](https://user-images.githubusercontent.com/59403296/229353028-809e05d2-4887-40b1-b365-54e7317a34eb.jpg)

![-6c74d3ec572fac26.jpg](https://user-images.githubusercontent.com/59403296/229353031-2c4ad2db-8f09-4434-9f0b-61080079ed75.jpg)

![-2f273953177bc342.jpg](https://user-images.githubusercontent.com/59403296/229353030-e27a4b4f-5cce-4abf-b893-c1dd0a413f79.jpg)

![3175d72c8b5e15af.jpg](https://user-images.githubusercontent.com/59403296/229353029-0f24b9cb-4ad6-4cd4-8ef7-ad776e8931d9.jpg)

![-3dfa7ffad0c46acb.jpg](https://user-images.githubusercontent.com/59403296/229353027-210bc52b-e6c5-4c74-9485-aa179fa4543f.jpg)

![7f2463f6ae8afca2.jpg](https://user-images.githubusercontent.com/59403296/229353032-643dd532-0746-4256-83d4-9a2c8d407cac.jpg)

**Comment by MyoungXUE** [comment]
Thank you for your feedback, this is a bug in the Zeus CLI. We will solve this problem as soon as possible.
